### PR TITLE
[MM-31791] Support Packet Generation FRONTEND

### PR DESCRIPTION
**Pull request by hahmadia**
### Note that currently the download button won't work since the backend PR has not yet been merged!
Backend PR: https://github.com/mattermost/mattermost-server/pull/16667
Redux PR: https://github.com/mattermost/mattermost-redux/pull/1331

#### Summary

Frontend Support Packet Generation Modal

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-31791

#### Screenshots

image
image

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -2545,6 +2545,9 @@
   "combined_system_message.removed_from_team.one_you": "You were **removed from the team**.",
   "combined_system_message.removed_from_team.two": "{firstUser} and {secondUser} were **removed from the team**.",
   "combined_system_message.you": "You",
+  "commercial_support.download_support_packet": "Download Support Packet",
```
Comment (sbishel) [inline i18n/en.json:2548]: commercial_support.description missing

**Review by sbishel — APPROVED** [review]

File: i18n/en.json (hunk 1 above)
Comment (hahmadia) [reply i18n/en.json:2549]: Thanks for that catch. Seems that if I didn't pass the string directly (as oppose to storing it in a variable and passing it), the `make i18n-extract` wouldn't pick it up. Glad you caught it! Thanks :)

**Review by michaelgamble — CHANGES_REQUESTED** [review]
Please do the following changes:
- re-use an existing component for the button
- reuse an existing component for the info field
- update the info areas content to match whats in the figma design (we updated this together in our zoom)

**Review by wiersgallak — APPROVED** [review]
Looks good to me.  Thanks @hahmadia

**Review by michaelgamble — APPROVED** [review]
Looks good :)

**Review by furqanmlk — APPROVED** [review]
All looks good ✅ 

[Test case created] (https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/design?projectId=10302)

**Pull request merged**